openmetadata-ui/src/main/resources/ui/src/components/DataModels/DataModelDetails.component.tsx

```diff
@@ -313,15 +314,14 @@ const DataModelDetails = ({
         ),
         key: EntityTabs.LINEAGE,
         children: (
-          <Card
-            className="card-body-full m-md w-auto h-60vh"
-            data-testid="lineage-details">
+          <div className="h-60vh">
             <EntityLineageComponent
               deleted={deleted}
+              entity={dataModelData as SourceType}
```

Comment:
Because, every entity details page stores data into there respective formats like Table,Pipeline, Topic etc. 
The ultimate purpose of introducing this param is to fetch breadcrumbs and for that we are using a common method `getEntityBreadcrumbs` which accepts param of type SourceType, hence it is typecasted to it.